helpers/keyCodes.js

```diff
@@ -0,0 +1,10 @@
+export const DOWN = 'ArrowDown';
```

Comment:
For the currently written exports, it should be possible to do:
```
export const
    DOWN = 'ArrowDown',
    END = 'End',
    ENTER = 'Enter',
    ESC = 'Escape',
    HOME = 'Home',
    LEFT = 'ArrowLeft',
    RIGHT = 'ArrowRight',
    SPACE = 'Space',
    TAB = 'Tab',
    UP = 'ArrowUp';
```
However, I am a bit on the fence in terms of exporting these unqualified constants.  The chance of collision is low, but certainly not zero.  Another approach, which I kind of prefer is:
```
export const keyCodes = {
    DOWN: 'ArrowDown',
    END: 'End',
    ENTER: 'Enter',
    ESC: 'Escape',
    HOME: 'Home',
    LEFT: 'ArrowLeft',
    RIGHT: 'ArrowRight',
    SPACE: 'Space',
    TAB: 'Tab',
    UP: 'ArrowUp'
};

import { keyCodes } from '../../helpers/keyCodes.js';
const { LEFT, RIGHT } = keyCodes;
```
The destructuring statement being optional for the consumer.  It gives the consumer the choice on how they want to reference the key codes.

Reply:
> The chance of collision is low, but certainly not zero

You can rename imports as well, no?

```js
import { LEFT as KeyLeft } from '../../helpers/keyCodes.js';
```

The separate exports is one of the big-benefits of ESM, and is better for dead code removal.

Reply:
Yeah that's a fair point.  Perhaps we should just qualify the names like that by default.  That is, `KeyLeft` vs `LEFT`.

Reply:
If we're just using the straight up name for the key, what's the difference between just using `e.code === 'KeyLeft'`? The original point of the module was to hide the magic keyCode constants but exporting them as their actual code seems silly to me.

Reply:
That's a good question - previously there was value when we were using the numeric codes with `keyCode`.  If we are going to use `key` then I don't see a lot of value here, but I think we need to figure which API we need to use (previously it was `keyCode`) - `keyCode` (deprecated), `code` (recommended by Mozilla but possibly has issues with mappings), or `key` (which was previously not used for some reason).  If necessary, we can then update other components that currently use `keyCode`.

My intent in the previous comment wasn't to just export the same thing - I actually didn't really pay attention to the actual values.  Rather, was just wanting to avoid collision and choose default names that clearly represent "key"s, avoiding rename in the import statement and addressing the somewhat minor concern of collision.

Reply:
I was more partial towards the object as well but basically what Owen mentioned is main reason I decided against it.

So I still prefer to have the constants since then I don't have to look up if it is `Esc` or `Escape` or maybe it is both ahh! 

So I don't like the preface of key on every key. We are importing from keyCodes we know they are keys. However, I hate all caps constants so having `keyLeft` is better for me.

Reply:
So this discussion about keys is in two places:
>So I think this is growing bigger then this PR and should be thought more about before any patterns are made in core about this. I found that Margaree has fixed my edge problems with tests. So I am going to copy what Calendar did.